src/frontend/main.go

```diff
@@ -142,6 +149,7 @@ func main() {
 	r.PathPrefix("/static/").Handler(http.StripPrefix("/static/", http.FileServer(http.Dir("./static/"))))
 	r.HandleFunc("/robots.txt", func(w http.ResponseWriter, _ *http.Request) { fmt.Fprint(w, "User-agent: *\nDisallow: /") })
 	r.HandleFunc("/_healthz", func(w http.ResponseWriter, _ *http.Request) { fmt.Fprint(w, "ok") })
+	r.HandleFunc("/metrics", prometheusExporter.ServeHTTP).Methods(http.MethodGet, http.MethodHead)
```

Comment:
We might want to add some error handling around this as it's possible that `prometheusExporter` hasn't initialised successfully